scss/_patterns_pull-quotes.scss

```diff
@@ -1,61 +1,116 @@
 @import 'settings';
 
-@mixin vf-quotes {
-  color: $color-mid-light;
+@mixin vf-quote-mark {
+  color: $color-mid-dark;
   display: inline-block;
-  font-size: 3rem;
   font-weight: bold;
-  max-width: 1.25rem;
   position: absolute;
+}
 
-  @media (max-width: $breakpoint-heading-threshold) {
-    font-size: 2.75rem;
-  }
+@mixin vf-pull-quote {
+  border: 0;
+  font-size: pow($ms-ratio, 2) * 1rem;
+  margin: $spv-inter--scaleable 0 $spv-inter--scaleable + $sp-unit;
+  overflow: visible;
+  position: relative;
 }
 
 // Pull quote styling
 @mixin vf-p-pull-quotes {
   .p-pull-quote {
-    border: 0;
-    margin: $spv-inter--scaleable 0 $spv-inter--scaleable + $sp-unit;
-    overflow: visible;
     padding: 0 $sph-inter * 2;
-    position: relative;
 
-    > p {
+    &__quote {
+      @extend %vf-heading-4;
```

Comment:
It doesn't really make sense to extend `%vf-heading-4` here then override some values for small and large variants. Why not just extend `%vf-heading-3` for large, `%vf-heading-4` for default, and `%default-text` for small?

Reply:
That is what's happening...

There's no overriding of %vf-heading-4 styles by the other two, they're selected by their full class name.

Reply:
%default-text doesn't do the job, that's the main problem. I'll see I if can make it tidier.